README.md

```diff
@@ -84,42 +100,57 @@ export default class StepPlanet extends Steps {
 }
 ```
 
-The inheritance of the `Steps` class expose following class fields:
+The manager exposed his options to each steps, they can access it with `this.options`.
+The inheritance of the `Steps` class exposes following the class fields:
+
+#### `id`
+
+`String`
+
+The route identifier, an alias to used inside the app instead of the route.
+
+_The parameter is a public instance fields._
 
 #### `route`
 
 `String`
 
-Route for step navigation. `StepManager` use hash for steps navigation.
+The route for the step navigation. `StepManager` use hash for steps navigation (`#people`).
+
+_The parameter is a public instance fields._
 
 #### `selector`
 
 `String`
 
-The CSS selector use in the template to identify the step.
+The CSS selector used in the template to identify the step.
+
+_The parameter is a public instance fields._
 
 #### `optionalStep`
 
 `Boolean`
 
-Declare if the step is optional and can be submit without validation. The parameter is a public instance fields like `route` or `selector`.
+Declare if the step is optional and can be submit without validation. The validation is set on the `canTheStepBeDisplayed` function.
```

Comment:
_To declare_ ?

J'ai l'impression qu'en ajoutant un "To" devant ces définitions, cela permet davantage de dire "ça sert à...".

Reply:
Oui tu as raison, c'est mieux pour cette phrase.
Pour les autres ça me choque moins.